uranus flat field e2e is failing

**Describe the bug**
Run the flat field e2e test and the uranus flat field crashes with:

```
tests/e2e_tests/flatfield_e2e.py:422: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
corgidrp/walker.py:526: in run_recipe
    curr_dataset = step_func(curr_dataset, *other_args, **kwargs)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
corgidrp/flat.py:323: in create_onsky_flatfield
    resi_images_dataset=flatfield_residuals(raster_images_dataset,N=N)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
corgidrp/flat.py:95: in flatfield_residuals
    images_split = np.array(np.split(np.array(raster_images),N))

E               ValueError: array split does not result in an equal division

../../miniconda3/envs/corgidrp/lib/python3.12/site-packages/numpy/lib/_shape_base_impl.py:877: ValueError
```

**To Reproduce**
Ran flatfield_e2e.py with default parameters

**Expected behavior**
I think we may need to set N=1 as the default. To investigate. 

**corgidrp version:**
 - develop

**Additional context**
Add any other context about the problem here.

--- a/corgidrp/flat.py
+++ b/corgidrp/flat.py
@@ -238,7 +238,7 @@ def combine_flatfield_rasters(resi_images_dataset,cent=None,planet=None,band=Non
     return (full_residuals,err_residuals,cent_n)
     
     
-def create_onsky_flatfield(dataset, planet=None,band=None,up_radius=55,im_size=None,N=3,rad_mask=None, planet_rad=None, n_pix=44, n_pad=None, sky_annulus_rin=2, sky_annulus_rout=4,image_center_x=512,image_center_y=512):
+def create_onsky_flatfield(dataset, planet=None,band=None,up_radius=55,im_size=None,N=1,rad_mask=None, planet_rad=None, n_pix=44, n_pad=None, sky_annulus_rin=2, sky_annulus_rout=4,image_center_x=512,image_center_y=512):
     """Turn this dataset of image frames of uranus or neptune raster scannned that were taken for performing the flat calibration and create one master flat image. 
     The input image frames are L2b image frames that have been dark subtracted, divided by k-gain, divided by EM gain, desmeared. 
 

